sharpy/aero/utils/airfoilpolars.py

```diff
@@ -62,7 +61,7 @@ def get_coefs(self, aoa_deg):
         cd = self.cd_interp(aoa_deg)
         cm = self.cm_interp(aoa_deg)
 
-        return cl, cd, cm
+        return cl[0], cd[0], cm[0]
```

Comment:
Fixed one of the pesky errors converting an array to a scalar during the unit test - this seems to work fine.